wise.html

```diff
@@ -0,0 +1,134 @@
+                    </a>
+                </div>
+
+                <div class="app-google">
+                    <a href="#">
+                        <img src="Google_Play_Store_badge_EN.svg.png" alt="image-app-google" width="210px" height="60px">
+                    </a>
+                </div>
+            </div>
+
+        </div>
+
+        <div class="sect2">
+            <p>Você envia exatamente<p>
+            <div class="form1">
+                <form action="">
+                <input type="text" value="1000">
+                    <a href="#">
+                        <img class="wds-flag wds-flag-gbp" src="https://wise.com/web-art/assets/flags/gbp.svg" alt width="22" height="22">
+                        <label for="number">GBP</label> 
+                        <i class="fa fa-caret-down" style="font-size:15px"></i>
+                    </a>
+                </form>
+            </div>
+            <div class="calc">
+                <div class="calc-1">
+                    <ul>
+                        <li>&nbsp;<i class="fa fa-circle" style="font-size:14px; color: #b9bdb5"></i>&nbsp;&nbsp;&nbsp; 0 GBP</li>
+                        <li>&nbsp;<i class="fa fa-circle" style="font-size:14px; color: #b9bdb5"></i>&nbsp;&nbsp;&nbsp; 4,08 GBP</li>
+                        <li><i class="fa fa-minus-circle" style="font-size:24px; color: #b9bdb5"></i>&nbsp;&nbsp; 4,08 GBP</li><br>
+                        <li><i class="fa fa-times-circle" style="font-size:24px; color: #b9bdb5"></i>&nbsp;&nbsp; 995,92 GBP</li>
+                        <li class="calc-1-2"><i class="fa fa-times-circle" style="font-size:24px; color: #b9bdb5"></i>&nbsp&nbsp;&nbsp;<a href="#">1,1832</a></li>
+                    </ul>
+                </div>
+                <div class="calc-2">
+                    <ul>
+                        <li class="calc-2-3"><a href="#">Tarifa de transferência bancária <i class="fa fa-caret-down" style="font-size:15px"></i></a></li>
+                        <li>Nossa tarifa</li>
+                        <li>Total em tarifas</li><br>
+                        <li>Valor que converteremos</li>
+                        <li>Câmbio garantido</li>
+                    </ul>
+                </div>
+            </div>
+            <div class="sect3">
+                <form action="">
+                <input type="text" value="1178,41">
+                    <a href="#">
+                        <img class="wds-flag wds-flag-eur" src="https://wise.com/web-art/assets/flags/eur.svg" alt width="22" height="22">
+                        <label for="number">EUR</label> 
+                        <i class="fa fa-caret-down" style="font-size:15px"></i>
+                    </a>
+                </form>
+            </div>
+            <div class="main-info">
+                <p class="info">You could save up to <b>38,89 GBP</b></p>
+                <p class="info">Should arrive <b>em 11 horas</b></p>
+            </div>
+            <div class="last-btn">
```

Comment:
@GenniferSebastiao 
on css, just for coding refactoring purpose  ,apply nth-child 
like 
.last-btn:nth-child(1){
}
.last-btn:nth-child(2){
}